client/blocks/importer/blogger/index.tsx

```diff
@@ -84,13 +83,8 @@ export const BloggerImporter: React.FunctionComponent< ImporterBaseProps > = ( p
 	}
 
 	function onSiteViewClick() {
-		if ( isEnabled( 'onboarding/import-redirect-to-themes' ) ) {
```

Comment:
I think this feature flag has been enabled 16 months ago. I also don't see any `calypso_site_importer_view_site` usage in our trends. So it seems totally safer to remove.